src/hooks/web3/useBridgeSommToEthTx.tsx

```diff
@@ -161,49 +101,107 @@ export const useBridgeSommToEthTx = () => {
           },
         },
       })
-      if (res.code !== 0) {
-        analytics.track("bridge.contract-failed", {
```

Comment:
It's better to exit early here as a guard condition https://en.wikipedia.org/wiki/Guard_(computer_science)